test/box/gh-5304-insert_during_recovery.test.lua

```diff
@@ -18,6 +18,16 @@ test_run:cmd('start server test with args="replace" with crash_expected=True')
 test_run:cmd('start server test with args="insert" with crash_expected=True')
 test_run:cmd('start server test with args="upsert" with crash_expected=True')
 test_run:cmd('start server test with args="replace is_recovery_finished"')
+test_run:cmd('switch test')
+box.space.temp:select()
+box.space.loc:select()
+-- Creating a new space and index invoke on_replace trigger in _index space,
+-- which insert tuples in temp and loc spaces (see gh-5304-insert_during_recovery.lua)!
+s0 = box.schema.space.create('test')
+i0 = s0:create_index('test')
+box.space.temp:select()
```

Comment:
Could you also add selects to other (insert/update) test cases to make sure that on_replace trigger was called?